entrypoint.sh

```diff
@@ -13,4 +13,6 @@ if [ ! -r "$HOME/.abuild/abuild.conf" ]; then
   abuild-keygen -n -i -a
 fi
 
+sudo apk -U upgrade -a
```

Comment:
Why would `upgrade` need the `-a` flag?
Is `upgrade` what we want? I would think `update` would be best.

Reply:
update only updates the indexes. If you dont upgrade the system you will be building against packages installed when docker image was generated. --available will always make sure you packages match the ones in the index, else it will only do a check for pkgname with an increased pkgver-pkgrel.